[2024-12-19T08:24:41.940+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T08:24:41.952+0530] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: movie_pipeline.scrape_collection_page manual__2024-12-19T02:54:05.172476+00:00 [queued]>
[2024-12-19T08:24:41.960+0530] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: movie_pipeline.scrape_collection_page manual__2024-12-19T02:54:05.172476+00:00 [queued]>
[2024-12-19T08:24:41.961+0530] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-19T08:24:41.982+0530] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): scrape_collection_page> on 2024-12-19 02:54:05.172476+00:00
[2024-12-19T08:24:41.990+0530] {logging_mixin.py:190} WARNING - /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=7626) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T08:24:41.991+0530] {standard_task_runner.py:72} INFO - Started process 7652 to run task
[2024-12-19T08:24:41.992+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'movie_pipeline', 'scrape_collection_page', 'manual__2024-12-19T02:54:05.172476+00:00', '--job-id', '200', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/var/folders/h6/l2cjrfls77b6_h6zs5w5vn4r0000gn/T/tmpwrfhinpl']
[2024-12-19T08:24:41.996+0530] {standard_task_runner.py:105} INFO - Job 200: Subtask scrape_collection_page
[2024-12-19T08:24:42.051+0530] {task_command.py:467} INFO - Running <TaskInstance: movie_pipeline.scrape_collection_page manual__2024-12-19T02:54:05.172476+00:00 [running]> on host Jayeshs-MacBook-Air.local
[2024-12-19T08:24:42.134+0530] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='movie_pipeline' AIRFLOW_CTX_TASK_ID='scrape_collection_page' AIRFLOW_CTX_EXECUTION_DATE='2024-12-19T02:54:05.172476+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-19T02:54:05.172476+00:00'
[2024-12-19T08:24:42.137+0530] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-19T08:24:42.137+0530] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-19T08:24:42.138+0530] {logging_mixin.py:190} INFO - Current task name:scrape_collection_page state:running start_date:2024-12-19 02:54:41.953294+00:00
[2024-12-19T08:24:42.138+0530] {logging_mixin.py:190} INFO - Dag name:movie_pipeline and current dag run status:running
[2024-12-19T08:24:42.139+0530] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T08:24:44.051+0530] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/src/extract.py", line 31, in page_extract
    with open(path,'w') as f:
         ^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'run_20241219/boxoffice_html_20241219.json'
[2024-12-19T08:24:44.085+0530] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-12-19T08:24:44.086+0530] {logging_mixin.py:190} INFO - Task start:2024-12-19 02:54:41.953294+00:00 end:2024-12-19 02:54:44.084312+00:00 duration:2.131018
[2024-12-19T08:24:44.088+0530] {logging_mixin.py:190} INFO - Task:<Task(PythonOperator): scrape_collection_page> dag:<DAG: movie_pipeline> dagrun:<DagRun movie_pipeline @ 2024-12-19 02:54:05.172476+00:00: manual__2024-12-19T02:54:05.172476+00:00, state:running, queued_at: 2024-12-19 02:54:05.200898+00:00. externally triggered: True>
[2024-12-19T08:24:44.088+0530] {logging_mixin.py:190} INFO - Failure caused by [Errno 2] No such file or directory: 'run_20241219/boxoffice_html_20241219.json'
[2024-12-19T08:24:44.089+0530] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=movie_pipeline, task_id=scrape_collection_page, run_id=manual__2024-12-19T02:54:05.172476+00:00, execution_date=20241219T025405, start_date=20241219T025441, end_date=20241219T025444
[2024-12-19T08:24:44.191+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T08:24:44.192+0530] {standard_task_runner.py:124} ERROR - Failed to execute job 200 for task scrape_collection_page ([Errno 2] No such file or directory: 'run_20241219/boxoffice_html_20241219.json'; 7652)
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/src/extract.py", line 31, in page_extract
    with open(path,'w') as f:
         ^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'run_20241219/boxoffice_html_20241219.json'
[2024-12-19T08:24:44.242+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-12-19T08:24:44.282+0530] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T08:24:44.284+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
